server/views/pages/activities/manage-schedules/view-activity.njk

```diff
@@ -228,6 +229,28 @@
                     }
                 }), dateAndScheduleRows) %}
             {% endif %}
+            {% else %}
+            {% set dateAndScheduleRows = (dateAndScheduleRows.push({
+                key: {
+                    text: "End date"
+                },
+                value: {
+                    text: "None set"
+                },
+                actions: {
+                    items: [
+                        {
+                            href: "/activities/create/end-date?fromEditActivity=true&preserveHistory=true",
+                            text: "Change",
+                            visuallyHiddenText: "change end date",
+                            classes: "govuk-link--no-visited-state"
+                        }
+                    ]
+                }
+            }), dateAndScheduleRows) %}
+            {% endif %}
```

Comment:
Minor – but I think this would be neater as inline conditionals rather than a new code block. Looks like it's just a couple of lines of text that are different?